pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_uploader_tool.adoc

```diff
@@ -0,0 +1,66 @@
+[id="uploader_tool"]
```

Comment:
s / uploader_tool / uploading-content-with-the-uploader-tool_{context}

Reply:
Done